client/src/components/Profile/Profile.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../../css/Profile.css';
+import blankProfilePicture from '../../assets/blank-profile-picture.png';
+import * as Routes from '../../routes';
+import { Redirect } from 'react-router-dom';
+
+/**
+ * Displays the Profile icon in the navbar + dropdown components for logout and viewing profile
+ * @param {func} logout connects to backend to end a users current session and log them out.
+ */
+const Profile = ({ logout }) => {
+  const [redirectToMemberPage, setRedirectToMemberPage] = React.useState(false);
+
+  const handleProfileRedirect = () => {
+    setRedirectToMemberPage(true);
+  };
+
+  return (
+    <div className="dropdown">
+      {redirectToMemberPage ? (
+        <Redirect to={Routes.MEMBER_PAGE_ROUTE} />
+      ) : (
+        <></>
```

Comment:
you can do `{redirectToMemberPage && <Redirect to {Routes.MEMBER_PAGE_ROUTE} />` so you don't have to render an empty div